src/Microsoft.Dnx.Tooling/Publish/PublishPackage.cs

```diff
@@ -30,11 +31,13 @@ public void Emit(PublishRoot root)
             {
                 NuGetPackage = srcNupkgPath,
                 SourcePackages = root.TargetPackagesPath,
-                Reports = root.Reports
             };
 
+            // Mute "packages add" subcommand
+            options.Reports = Reports.Constants.NullReports;
```

Comment:
When publishing a starter web app, there are 160+ packages get published and it means that this method is going to be called many many times. That's why I create a constant to avoid allocating/deallocating numerous `NullReport` objects.